docs/cheat-sheet.rst

```diff
@@ -289,11 +289,13 @@ The *investments* dataset provides a separate way to look at making R&D investme
   Column Name  Data Type  Description                                                                                            
   ============ ========== =====================================================================================================
   Investment   String     Name of the R&D investment. Distinct from the Scenarios.                                               
-  Category     String     Names of the R&D categories being invested in. Within each row, the Category must match the Tranche.   
-  Tranche      String     Names of the tranches within the Investment. Within each row, the Tranche must match the Category
+  Category     String     Names of the R&D categories being invested in. Within each row, the Category must match the Tranche. The set of Categories in the *investments* dataset must match the set of Categories in the *tranches* dataset.
+  Tranche      String     Names of the tranches within the Investment. Within each row, the Tranche must match the Category. The set of Tranches in the *investments* dataset must match the set of Tranches in the *tranches* dataset.
   Notes        String     Additional user-defined information. Not used by Tyche.                                                
   ============ ================================================================================================================
 
+**Relationship between Categories, Tranches, Scenarios, and Investments.** Both the *designs* and *parameters* dataset contain technology data under multiple Scenarios. Each Scenario represents the technological outcomes from one or more Tranches, and each Tranche represents a unit of R&D investment in a single Category (or research area). Scenarios and their component Tranches are defined in the *tranches* dataset. Tranches can also be combined to form Investments, as defined in the *investments* dataset.
```

Comment:
This para is more about how Scenarios and Investments are defined, not about how the datasets themselves correspond. I revised accordingly and also removed the statement about Tranches being one-to-one with Scenarios, which isn't necessarily or even often true.